src/main/java/de/zalando/zmon/dataservice/data/QueryStoreConfig.java

```diff
@@ -0,0 +1,24 @@
+package de.zalando.zmon.dataservice.data;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+
+import de.zalando.zmon.dataservice.config.DataServiceConfigProperties;
+
+/**
+ * Created by mabdelhameed on 14/11/2017.
+ */
+@Configuration
+public class QueryStoreConfig {
+
+    @Bean
+    DataPointsQueryStore dataPointsQueryStore(DataServiceConfigProperties config) {
+        if (config.isDatapointsRedisEnabled()) {
```

Comment:
what about `config.isKairosdbEnabled()` ? Do we still support this option?

Reply:
We have it disabling the entire path here https://github.com/zalando-zmon/zmon-data-service/pull/76/files#diff-43732a200088b5d2aa03bcc45c512956R134